jsignature/fields.py

```diff
@@ -10,19 +10,17 @@
 
 from .forms import (
     JSignatureField as JSignatureFormField,
-    JSIGNATURE_EMPTY_VALUES)
+    JSIGNATURE_EMPTY_VALUES
+)
 
 
-class JSignatureField(six.with_metaclass(models.SubfieldBase, models.Field)):
+class JSignatureField(models.TextField):
```

Comment:
Django 1.10 no longer offers SubfieldBase. Recommended practice instead is to use from_db_value.